src/commands/alias/set.ts

```diff
@@ -106,6 +106,8 @@ export default async function set(
     return 1;
   }
 
+  output.warn(`The ${cmd('now alias')} command was deprecated in favour of ${cmd('now --target production')}.`);
```

Comment:
Have you made sure `now alias` (without arguments)  does not print this message twice? (they are using the same code path)